eng/build.yml

```diff
@@ -151,3 +149,33 @@ jobs:
           BuildConfig: $(_BuildConfig)
           BlobFeedUrl: $(PB_PublishBlobFeedUrl)
           PublishType: $(_PublishType)
+
+    - task: PublishTestResults@1	
```

Comment:
This should not be needed. It is done by arcade. Any reason why you are adding it back?

Reply:
Arcade doesn't give you meaningful names for the test results or the zip files.  See [here](https://dev.azure.com/dnceng/public/_build/results?buildId=185074&view=ms.vss-test-web.build-test-results-tab) for example:

![image](https://user-images.githubusercontent.com/145043/57558419-7b8e6780-7332-11e9-8eff-c7dce76e5600.png)

There's discussion of this on this arcade PR: https://github.com/dotnet/arcade/pull/2525.  I made a similar change to core-sdk here: https://github.com/dotnet/core-sdk/pull/1609